encord/common/time_parser.py

```diff
@@ -1,3 +1,15 @@
+"""
```

Comment:
This is an internal file, probably shouldn't go to docs

Reply:
Updated